### DISCUSSION TITLE
MISP to MySQL

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [MISP/PyMISP#775] [General]
Someone please help me with sample example to store MISP events to MySQL DB. I'm noob.

**Comment by Rafiot** [comment]
Not sure where to start, but have you looked at the MISP documentation: https://misp.github.io/MISP/

**Reply by Jackson-Pollock** [reply]
Yes! I'm clear with import but don't know how to store in db.

**Reply by Jackson-Pollock** [reply]
My MISP is ready I'm looking to store events to db using pymisp

**Reply by Rafiot** [reply]
You can look at this tutorial: https://github.com/MISP/PyMISP/blob/main/docs/tutorial/FullOverview.ipynb

**Reply by Jackson-Pollock** [reply]
Thanks
